benchexec/tablegenerator/react-table/scripts/start.js

```diff
@@ -40,6 +41,18 @@ const createDevServerConfig = require("../config/webpackDevServer.config");
 const useYarn = fs.existsSync(paths.yarnLockFile);
 const isInteractive = process.stdout.isTTY;
 
+const dataParam = process.argv[2];
+
+if (dataParam) {
+  const content = fs.readFileSync(path.resolve(__dirname, "../", dataParam));
+  fs.writeFileSync(path.resolve("./src/data/custom-data.json"), content);
+}
+
+const DATA =
+  (dataParam && "./src/data/custom-data.json") || "./src/data/data.json";
```

Comment:
A ternary operator is probably more obvious in such a case. And the value could be directly assigned to `process.env.DATA`.